cmd/kuberhealthy/Makefile

```diff
@@ -1,12 +1,12 @@
-# IMAGE="kuberhealthy/kuberhealthy"
-# TAG="2.2.0"
+IMAGE="kuberhealthy/kuberhealthy"
+TAG="2.2.0"
 
 build:
-	docker build -t kuberhealthy/kuberhealthy:v2.2.2 -f Dockerfile ../../
-
+	docker build -t ${IMAGE}:${TAG} -f Dockerfile ../../
+build-dev:
+	docker build -t kuberhealthy:dev -f Dockerfile ../../
 push:
-	docker push kuberhealthy/kuberhealthy:v2.2.2
-
+	docker push kuberhealthy/${IMAGE}:${TAG}
```

Comment:
You don't need the extra `kuberhealthy/` here because it's included in your `IMAGE` var

Reply:
nice catch